src/ResourceManager/StorageSync/devops.ps1

```diff
@@ -0,0 +1,20 @@
+$ErrorActionPreference = "Stop"
```

Comment:
@afedyashov what is the purpose of this script?

Reply:
loader for our devops module - which is loaded via $env:psuserpreferences as per what's documented in Repo-Tasks.psm1 (below is from that file):
We allow users to include any helper powershell scripts they would like to include in the current session
Currently we support two ways to include helper powershell scripts
1) psuserspreferences environment variable
2) $env:USERPROFILE\psFiles directory
We will include all *.ps1 files from any of the above mentioned locations

Reply:
This comment will also apply to the `storageSyncDevops.psm1` file below: it's not necessary to check-in scripts that you use for building/debugging/testing, but if you feel that they will be helpful to others, feel free to keep them checked-in, but possibly create a new folder to put all of these scripts and tools in so they don't get mixed up with the regular cmdlet project and scripts

Reply:
Would you please define how critical this comment is? I find it hard to justify removal of tools that keeps developers productive. All of this tools are StorageSync-specific and incorporate things like either build routines from the guide and things I discovered myself such as static analysis log parsings, regeneration of help md files, etc. If new developer joins the project from our side - first thing they need to learn is devops capabilites from these scripts, rather than learning on their own various scripts doing various things in the repository. StorageSync folder will grow as we migrate all of our cmdlets into AzureRm, and command that we run to build, troubleshoot failed builds, generate documentation, etc - all will be placed in those modules, and they belong to StorageSync folder beccause they are specific to strorage sync needs.